packages/mobx/src/errors.ts

```diff
@@ -3,6 +3,8 @@ const niceErrors = {
     1(annotationType, key: PropertyKey) {
         return `Cannot apply '${annotationType}' to '${key.toString()}': Field not found.`
     },
+    3: "'ownKeys()' can only be used on observable objects",
```

Comment:
NIT: if these numbers wheren't here yet, this were error codes used in the past, so it is better not to recycle them in case someone is looking for them (in which case they need to go back in history of this file, I guess we should have left a comment when removing, but alas)

Reply:
I will take a look if I can revive these removed in previous PRs (not sure if I hadn't already repurposed some).
What's the recommended practice here? Keeping the error as is and providing comment that it's no longer used?

Reply:
Just leaving them in the source code, but commented out so that they don't increase bundles unnecessarily. Hopefully the remain at least searchable in GH history and issues if the numbers are not recycled. I have no idea if that actually ever happens, so don't worry too much about it :)